packages/playwright-core/src/server/chromium/crServiceWorker.ts

```diff
@@ -51,6 +53,29 @@ export class CRServiceWorker extends Worker {
       // Resume service worker after restart.
       session._sendMayFail('Runtime.runIfWaitingForDebugger', {});
     });
+    session.on('Runtime.consoleAPICalled', event => {
+      if (event.executionContextId === 0) {
+        // DevTools protocol stores the last 1000 console messages. These
+        // messages are always reported even for removed execution contexts. In
```

Comment:
How can the context be removed for a service worker where it is the only context?